test/e2e/upgradetest/upgrade_test.go

```diff
@@ -34,7 +34,10 @@ func TestResize(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	time.Sleep(10 * time.Second)
+	if err = k8sutil.WaitEtcdTPRReady(testF.KubeCli.CoreV1().RESTClient(), 3*time.Second, 30*time.Second, testF.KubeNS); err != nil {
```

Comment:
It takes about 20 seconds from the time of creating the operator to the TPR to being visible at the api server. So a 3s interval check for a total of 30s seemed fine. The interval doesn't matter as much as the total timeout.